timezone/src/include/timezone/dialogs.rb

```diff
@@ -572,22 +573,21 @@ def TimezoneDialog(args)
         # true by default (fate#303520)
         @ntp_used = true
         # configure NTP client
-        Builtins.srandom
-        @ntp_server = Builtins.sformat(
-          "%1.opensuse.pool.ntp.org",
-          Builtins.random(4)
-        )
+        # to prevent misusage of ntp.org we need to distinguish opensuse and SLE usage
+        base_products = Product.FindBaseProducts
```

Comment:
Hm, this `Product` module initializes libzypp which might take some time (refreshing the services or repositories). 

What about using the simpler [OSRelease](https://github.com/yast/yast-yast2/blob/master/library/general/src/modules/OSRelease.rb) module instead? That just parses the `/etc/os-release` without any overhead.
And that file is also present in the inst-sys so it will work correctly also during the installation.

Reply:
I see, if it is used only in the installation then it's OK.